cmd/semv/main.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"html/template"
+	"os"
+	"strings"
+
+	"github.com/rancher/ecm-distro-tools/release/semver"
+	"gopkg.in/yaml.v3"
+)
+
+func Format(v *semver.Version, format string) (string, error) {
```

Comment:
```suggestion
func format(v *semver.Version, format string) (string, error) {
```

Can you move this to just above main so we can have the boilerplate and setup stuff right at the top?